src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -1059,6 +1059,26 @@
       }
     }
   },
+  "coach-indicator-collapsed-ring-rounding-increment": {
+    "value": "6px",
+    "type": "number",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "coach-indicator-collapsed-ring-rounding-increment"
+      }
+    }
+  },
+  "coach-indicator-expanded-ring-rounding-increment": {
+    "value": "14px",
+    "type": "number",
```

Comment:
@lynnhao This should be "type": "spacing" or "type": "sizing". These tokens have a pixel value and therefor are _not_ Type: `number`.